supervision/annotators/core.py

```diff
@@ -1266,6 +1331,7 @@ def __init__(
         text_position: Position = Position.TOP_LEFT,
         color_lookup: ColorLookup = ColorLookup.CLASS,
         border_radius: int = 0,
+        smart_positions: bool = False,
```

Comment:
Let's drop `s` and rename it to `smart_position` as we have other arguments like `text_position`.